tests/lib/rules/no-misleading-character-class.js

```diff
@@ -148,24 +148,14 @@ ruleTester.run("no-misleading-character-class", rule, {
                 suggestions: [{ messageId: "suggestUnicodeFlag", output: "var r = /\\uDC4D[\\uD83D\\uDC4D]/u" }]
             }]
         },
-        {
-            code: "var r = /[👍]/",
-            languageOptions: { ecmaVersion: 3, sourceType: "script" },
-            errors: [{
-                column: 11,
-                endColumn: 13,
-                messageId: "surrogatePairWithoutUFlag",
-                suggestions: null // ecmaVersion doesn't support the 'u' flag
-            }]
-        },
         {
             code: "var r = /[👍]/",
             languageOptions: { ecmaVersion: 5, sourceType: "script" },
             errors: [{
                 column: 11,
                 endColumn: 13,
                 messageId: "surrogatePairWithoutUFlag",
-                suggestions: null // ecmaVersion doesn't support the 'u' flag
+                suggestions: null // ecmaVersion doesn't support the 'u' flag, ecmaVersion doesn't support the 'u' flag
```

Comment:
Is this change intentional?

Reply:
Oops, removed this change.

```suggestion
                suggestions: null // ecmaVersion doesn't support the 'u' flag
```